dev/deps/spark-deps-hadoop-2.7-hive-2.3

```diff
@@ -241,5 +241,6 @@ xml-apis/1.4.01//xml-apis-1.4.01.jar
 xmlenc/0.52//xmlenc-0.52.jar
 xz/1.5//xz-1.5.jar
 zjsonpatch/0.3.0//zjsonpatch-0.3.0.jar
-zookeeper/3.4.14//zookeeper-3.4.14.jar
+zookeeper-jute/3.6.2//zookeeper-jute-3.6.2.jar
```

Comment:
It seems we need zookeeper-jute-3.6.2.jar, otherwise:
```
21/01/13 22:49:56,890 ERROR [Driver] server.HiveServer2:186 : Unable to create a znode for this server instance
java.lang.Exception: Max znode creation wait time: 120s exhausted
	at org.apache.hive.service.server.HiveServer2.addServerInstanceToZooKeeper(HiveServer2.java:183)
	at org.apache.hive.service.server.HiveServer2.start(HiveServer2.java:128)
	at org.apache.spark.sql.hive.thriftserver.HiveThriftServer2.start(HiveThriftServer2.scala:230)
	at org.apache.spark.sql.hive.thriftserver.HiveThriftServer2$.main(HiveThriftServer2.scala:159)
	at org.apache.spark.sql.hive.thriftserver.HiveThriftServer2.main(HiveThriftServer2.scala)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.apache.spark.deploy.yarn.ApplicationMaster$$anon$2.run(ApplicationMaster.scala:732)
```